apps/fluent-tester/src/TestComponents/Button/ButtonTest.tsx

```diff
@@ -26,10 +27,13 @@ const buttonSections: TestSection[] = [
     name: 'Icon Button',
     component: ButtonIconTest,
   },
-  {
-    name: 'Toggle Button',
-    component: ToggleButtonTest,
-  },
+  Platform.select({
+    android: null, //Toggle Button is not implemented for Android
```

Comment:
Do we have a similar tester update needed for Compound button as well?

Reply:
No, it is not required since Compound button does not have a dedicated section, it just appears in other test sections